src/util.js

```diff
@@ -12,13 +12,58 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(internal) {
+(function(internal, testing) {
 
   function inherit(childCtor, parentCtor) {
     childCtor.prototype = Object.create(parentCtor.prototype);
     childCtor.prototype.constructor = childCtor;
   }
 
+  /*
```

Comment:
This is proper JSDoc, which I thought would be appropriate for this location. In Google we use the /** style comment for function documentation - do we want to stick with that or change to the other kind?